lib/tasks/schools_data.rake

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+namespace :schools_data do
+  HEADERS = %w[urn name town postcode lead_school open_date close_date].freeze
+  REFINED_CSV_PATH = Rails.root.join("data/schools.csv").freeze
+
+  # This task requires two csvs:
+  #  The establisment csv can be downloaded here https://get-information-schools.service.gov.uk/Downloads
+  #  under "Establishment fields"
+  #
+  #  Ask in slack for the lead schools csv as it isn't published anywhere
```

Comment:
Where good question...

Reply:
@defong Khez gave me a copy, but I didn't want to commit it into the repo since it changes yearly

Reply:
Can't find out how to upload a file outside of a document in confluence, putting it in google drive as that seems like a better fit